internal/namespaces/baremetal/v1/custom_offer_test.go

```diff
@@ -0,0 +1,16 @@
+package baremetal
+
+import (
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+)
+
+func Test_baremetalGetOffer(t *testing.T) {
+	offerID := "c5853302-63e4-40c7-a711-4a91629565c8"
+	t.Run("Simple", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw baremetal offer get offer-id=" + offerID,
```

Comment:
Could you make offer-id positional ?

Reply:
Yes, I can. Do we really need the offerID as an argument? Why not use the name of the offer instead? I don't see any cases where this would be a breaking change